### PR TITLE
cylc gsummary: update time from suite

### LINKED ISSUE #1310: cylc gsummary: always use suite last updated time for 'Updated' column
The `Updated` column in cylc gsummary should list the last updated time
for individual (running) suites, not the global gsummary updated time.

### PR DESCRIPTION
**Pull request by kaday**
Closes #1310

### CODE REVIEW

**Comment by kaday** [comment]
@benfitzpatrick please review

File: lib/cylc/gui/gsummary.py [hunk 1]
```diff
@@ -92,7 +91,12 @@ def get_task_cycle_statuses(host, suite, owner=None):
         task_order = task.split(".")
         task_order.append(status)
         task_cycle_statuses.append(tuple(task_order))
-    return task_cycle_statuses
+    for line in stdout.split("\n"):
```
Comment (benfitzpatrick) [inline lib/cylc/gui/gsummary.py]: You can search the whole multiline text (time + bracketed part) by using [re.M](https://docs.python.org/2/library/re.html#re.M)

File: lib/cylc/gui/gsummary.py [hunk 2]
```diff
@@ -789,10 +806,11 @@ def run(self):
                 self.prev_suites.remove((host, suite))
 
         # Get new information.
-        statuses, stop_summaries = get_new_statuses_and_stop_summaries(
+        statuses, stop_summaries, suite_update_time = (
```
Comment (benfitzpatrick) [inline lib/cylc/gui/gsummary.py]: Can it be "suite_update_times"?

**Comment by kaday** [comment]
@benfitzpatrick changes made and rebased

File: lib/cylc/gui/gsummary.py [hunk 3]
```diff
@@ -537,9 +541,9 @@ def _on_query_tooltip(self, widget, x, y, kbd_ctx, tooltip):
             host = model.get_value(parent_iter, 0)
             suite = model.get_value(parent_iter, 1)
             child_row_number = path[-1]
-        update_time = model.get_value(iter_, 4)
+        suite_update_times = model.get_value(iter_, 4)
```
Comment (benfitzpatrick) [inline lib/cylc/gui/gsummary.py]: Sorry, I meant `....times` just for the dictionary type variables in the `update` method and others - this one
is better as `....time`.

**Comment by kaday** [comment]
@benfitzpatrick variable name updated for the query tooltip and rebased

**Comment by benfitzpatrick** [comment]
Tested, looks good.

**Comment by benfitzpatrick** [comment]
Tested gpanel at the last minute, and it doesn't appear to work - needs fixing!

**Comment by kaday** [comment]
@benfitzpatrick I fixed it, sorry it took so long, both gsummary and gpanel work and have been tested.

**Pull request merged**